data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-consensual-creation-v1.0.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+
+    <changeSet author="R3.Corda" id="ledger-consensual-creation-v1.0">
+
+        <createTable tableName="consensual_transaction">
+            <column name="id" autoIncrement="true" type="BIGINT">
```

Comment:
I had assumed we were using the varchar representation of the txId as primary key. A numeric primary key might perform better to be fair, but we will definitely want to consider archiving and portability. Also, will need that txid here somewhere for select statements

Reply:
Ah, I thought we would just be generating an autoincrement sequence of arbitrary IDs for internal use, rather than coming from somewhere else. If we don't autogenerate it on insert, where does it come from? Are we talking about different IDs?

Reply:
This needs to be the transaction id (the root hash of the merkle tree)